src/components/Sidebar/Sidebar.js

```diff
@@ -1,31 +1,93 @@
 import React, { useState } from 'react';
-import { AccountCircle, Inbox, Mail } from '@material-ui/icons';
+import { AccountCircle, Inbox, Mail, MoreVert } from '@material-ui/icons';
 import IconButton from '@material-ui/core/IconButton';
 import MenuIcon from '@material-ui/icons/Menu';
-import { Divider, Drawer, List, ListItem, ListItemText } from '@material-ui/core';
+import {
+  Button,
+  Divider,
+  Drawer,
+  Hidden,
+  List,
+  ListItem,
+  ListItemText,
+  Menu,
+  MenuItem,
+} from '@material-ui/core';
 import useStyles from './SidebarStyles';
 import logo from '../../assets/app-logo.png';
 import { useGoogleAuth } from '../Auth/GoogleAuthProvider';
+import LoginModal from '../Auth/LoginModal';
 
 function Sidebar() {
   const classes = useStyles();
   const [isSideBarOpen, toggleSideBar] = useState(false);
-  const { googleUser } = useGoogleAuth();
+  const { isInitialized, isSignedIn, googleUser, signOut } = useGoogleAuth();
+  const [anchorEl, setAnchorEl] = React.useState(null);
```

Comment:
```suggestion
  const [anchorEl, setAnchorEl] = useState(null);
```

To keep it consistent with other hook usages.